src/PersonReader.java

```diff
@@ -1,6 +1,45 @@
+import java.io.File;
+import javax.swing.JFileChooser;
+import java.io.FileNotFoundException;
+import java.lang.String;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import javax.swing.filechooser.FileSystemView;
+
 public class PersonReader {
-    public static void main(String[] args) {
+    public static void main(String[] args) throws FileNotFoundException {
         // Write your code here ... ⬇
+        JFileChooser choose = new JFileChooser(FileSystemView.getFileSystemView());
```

Comment:
you could just do this too.
```suggestion
        JFileChooser choose = new JFileChooser();
```

Reply:
However, I would also add the following 2 lines
```java
File workingDirectory = new File(System.getProperty("user.dir")); // Set default directory to the project folder
chooser.setCurrentDirectory(workingDirectory);
```

They would make sure that the file chooser starts at the project directory rather then going all the way back to the computer home directory